internal/pkg/state/fixtures.go

```diff
@@ -50,6 +60,16 @@ func NewProjectSnapshot(s *State) (*fixtures.ProjectSnapshot, error) {
 		c.Rows = append(c.Rows, r)
 	}
 
+	schedules, err := testProject.SchedulerApi().ListSchedules()
+	if err != nil {
+		panic(err)
+	}
+	for _, schedule := range schedules {
+		configKey := model.ConfigKey{BranchId: defaultBranchId, ComponentId: model.SchedulerComponentId, Id: schedule.ConfigurationId}
+		scheduleConfig := s.MustGet(configKey).(*model.ConfigState).Remote
+		project.Schedules = append(project.Schedules, &fixtures.ScheduleConfig{Name: scheduleConfig.Name})
```

Comment:
@webrouse Tak mám to přidaný, akorát teď nevím jestli mi tam něco nechybí. 🤔 Neměl by mi ten test už failovat, když v `expected-state.json` nemám `schedules: [{ "name": "schedule1" }]`?

Reply:
pozriem to za chvilu, asi mal ked to nesedi :) mozes si tam skusit pridat `fmt.Println(project.Schedules)` a spustit iba ten jeden test, ... mal by si vidiet ci to nejake `Schedules` nacitalo.

Reply:
@JakubMatejka zatial to vyzera tak, ze testovacia cast je ok, ... upravil som `args` a pridal `--verbose` a nevidim tam ziadne `Scheduler API ` calls.

Reply:
ale callback sa vykona:
![image](https://user-images.githubusercontent.com/19371734/139866162-1ab46b64-0f56-47c3-8336-6b351dfa3e7b.png)
![image](https://user-images.githubusercontent.com/19371734/139866218-37e37c94-d1ad-4383-8c18-da1e312cb2bb.png)

Reply:
@JakubMatejka problem je tu:
![image](https://user-images.githubusercontent.com/19371734/139866794-8eb90a63-1230-44b4-8d1a-c00ce7ddca15.png)

Reply:
Nepouzivas `pool`, ktore mas v `u *UnitOfWork`, ale vytvaras nove, ktore nespustis.